hail/python/hail/utils/struct.py

```diff
@@ -166,6 +166,23 @@ def drop(self, *args):
         -------
         :class:`.Struct`
             Struct without certain fields.
+
+        Examples
+        --------
+
+        Define a Struct `s`
+
+         >>> s = hl.Struct(food=8, fruit=5, bar=2, apple=10)
```

Comment:
I think the alignment of the code examples with lines starting with `>>>` and the result shown below need to be aligned with the text above.